translations/ai_rules.tex/cs.po

```diff
@@ -84,7 +84,7 @@ msgid ""
 "They gain no benefits from any Fields.\n"
 "\n"
 msgstr ""
-"AI Hrdinové vždy \\textbf{automaticky vyhrají Boj} proti jakýmkoliv Neutrálním Jednotkám a zároveň \\textbf{Obsadí nebo Navštíví všechna Pole}, kterými procházejí.\n"
+"AI Hrdinové vždy \\textbf{automaticky vyhrají Souboj} proti jakýmkoliv Neutrálním Jednotkám a zároveň \\textbf{Obsadí nebo Navštíví všechna Pole}, kterými procházejí.\n"
```

Comment:
I'm not sure about the "Souboj proti" - "Souboj s" is more natural

Reply:
I'm not so sure about that either. It seems more natural, but it could also be misinterpreted to mean that if the AI heroes have neutral units with them, they automatically win the combat.

Reply:
Hmm. I mean, it could but it would be a very strange rule.

We could either leave it here with "proti" or rephrase the sentence to something like "...Soubor s jakymikoliv Neutralnimi Jednotkami, na ktere narazi..." 🤷 

Unfortunately this "souboj proti" is in multiple places.

Reply:
I would leave it with "proti".